.github/workflows/tests.yml

```diff
@@ -7,11 +7,16 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
@seratch I was not specifying the matrix os version as the `run-on` value, let me know if you want to use this approach which default to `ubuntu-latest` or the one found in the [python-sdk](https://github.com/slackapi/python-slack-sdk/commit/f2f3fd63e7b86abdb29a66f28b897402802ddafe)